cms/envs/common.py

```diff
@@ -74,6 +74,7 @@
     HEARTBEAT_CHECKS,
     HEARTBEAT_EXTENDED_CHECKS,
     HEARTBEAT_CELERY_TIMEOUT,
+    CELERY_CHECK_ROUTING_KEY,
```

Comment:
Can you rename this setting to be consistent with the other related settings?
Suggestion: `CELERY_CHECK_ROUTING_KEY` -> `HEARTBEAT_CELERY_ROUTING_KEY`